central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -63,6 +65,23 @@ const (
 	SELECT nf.Props_SrcEntity_Type, nf.Props_SrcEntity_Id, nf.Props_DstEntity_Type, nf.Props_DstEntity_Id, nf.Props_DstPort, nf.Props_L4Protocol, nf.LastSeenTimestamp, nf.ClusterId
 	FROM network_flows nf ` + joinStmt +
 		`WHERE nf.Props_DstEntity_Type = 1 AND nf.Props_DstEntity_Id = $1 AND nf.ClusterId = $2`
+
+	pruneStaleNetworkFlowsStmt = `DELETE FROM network_flows a USING (
+      SELECT MAX(flow_id) as max_flow, props_srcentity_type, props_srcentity_id, props_dstentity_type, props_dstentity_id, props_dstport, props_l4protocol, clusterid
```

Comment:
Even the db is not case sensitive, it is better to keep it consistent. Props_DstEntity_Type

Reply:
Two nit issues.